lookup-table.yaml

```diff
@@ -587,3 +587,15 @@
     - packageId: vram-guild.canvas
       loaders:
         - fabric
+- id: plasmo_voice
+  alternativeNames:
+    - plasmo-voice
+  tags:
+    - adventure
+    - utility
+    - miscellaneous
+  packages:
+    - packageId: plasmoapp.plasmo_voice
+      loaders:
+        - fabric
+        
```

Comment:
You should also make sure there are no spaces, else it doesn't count as a newline.